wsapi.go

```diff
@@ -267,9 +267,16 @@ type helloOp struct {
 // FailedHeartbeatAcks is the Number of heartbeat intervals to wait until forcing a connection restart.
 const FailedHeartbeatAcks time.Duration = 5 * time.Millisecond
 
+// HeartbeatLatency returns the latency between heartbeat acknowledgement and heartbeat send
+func (s *Session) HeartbeatLatency() time.Duration {
+
+	return s.LastHeartbeatAck.Sub(s.LastHeartbeatSent)
+
+}
+
 // heartbeat sends regular heartbeats to Discord so it knows the client
 // is still connected.  If you do not send these heartbeats Discord will
-// disconnect the websocket connection after a few seconds.
+// disconnect the websocket connection after a few seconds
```

Comment:
Sorry, last change, you should undo this change (comments should end in periods), you should also add a period in the comment for your change! Thanks so much.

Reply:
Will do